src/test/java/ca/bc/gov/backendstartapi/repository/FundingSourceRepositoryTest.java

```diff
@@ -0,0 +1,33 @@
+package ca.bc.gov.backendstartapi.repository;
+
+import ca.bc.gov.backendstartapi.entity.FundingSource;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase.Replace;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ExtendWith(SpringExtension.class)
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = Replace.NONE)
+class FundingSourceRepositoryTest {
+
+  @Autowired
+  private FundingSourceRepository fundingSourceRepository;
+
+  @Test
+  @DisplayName("findAllTest")
+  @Sql(scripts = {"classpath:scripts/FundingSourceRepositoryTest_findAll.sql"})
+  void findAllTest() {
+    List<FundingSource> sources = fundingSourceRepository.findAll();
+
+    Assertions.assertFalse(sources.isEmpty());
+    Assertions.assertEquals(3, sources.size());
+  }
```

Comment:
Do we need this test? It's just testing a third-party library here, not something we have written ourselves.

There might be a case in which we'd like to do it, but this doesn't look like one to me.

Reply:
I think we should have a test case to test both effective and expiry date. I've added two record, making them 5 instead of 3, but the find method should returns only three, the valid ones.